src/app/containers/Image/helpers/srcSetHelper.js

```diff
@@ -0,0 +1,29 @@
+import getIchefUrl from './ichefUrlHelper';
```

Comment:
I think you can omit the `Helper` from both `ichefUrlHelper` and `srcSetHelper` and just have `ichefUrl` and `srcset` as they're already in the `/helpers/` namespace.